apps/meteor/tests/end-to-end/api/25-teams.js

```diff
@@ -819,6 +819,77 @@ describe('[Teams]', () => {
 		});
 	});
 
+	describe('/teams.info', () => {
+		it('should successfully get a team info by name', (done) => {
+			request
+				.get(api('teams.info'))
+				.set(credentials)
+				.query({
+					teamName: publicTeam.name,
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.expect((response) => {
+					expect(response.body).to.have.property('success', true);
+					expect(response.body).to.have.property('teamInfo');
+					expect(response.body.teamInfo).to.have.property('_id').and.to.be.eq(publicTeam._id);
+					expect(response.body.teamInfo).to.have.property('name').and.to.be.eq(publicTeam.name);
+				})
+				.then(() => done())
+				.catch(done);
+		});
+		it('should successfully get a team info by id', (done) => {
+			request
+				.get(api('teams.info'))
+				.set(credentials)
+				.query({
+					teamId: publicTeam._id,
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.expect((response) => {
+					expect(response.body).to.have.property('success', true);
+					expect(response.body).to.have.property('teamInfo');
+					expect(response.body.teamInfo).to.have.property('_id').and.to.be.eq(publicTeam._id);
+					expect(response.body.teamInfo).to.have.property('name').and.to.be.eq(publicTeam.name);
```

Comment:
```suggestion
					expect(response.body.teamInfo).to.have.property('_id', publicTeam._id);
					expect(response.body.teamInfo).to.have.property('name', publicTeam.name);
```

You can check the property and its value all at once.

Reply:
Updated!